Library/Tracking/KSRAnalytics.swift

```diff
@@ -921,12 +933,13 @@ public final class KSRAnalytics {
   public func trackThanksPageViewed(
     project: Project,
     reward: Reward,
-    checkoutData: CheckoutPropertiesData?
+    checkoutData: CheckoutPropertiesData?,
+    typeContext: TypeContext
   ) {
     var props = projectProperties(from: project)
       .withAllValuesFrom(pledgeProperties(from: reward))
       // the context is always "newPledge" for this event
```

Comment:
Since the context is always `.newPledge` here I don't think we need to pass in the typeContext as a parameter anymore.